### ISSUE TITLE
Schema Error

### ISSUE THREAD

**Issue by Arttii** [confluentinc/kafka-connect-jdbc#15]
Hi,

I'm running into this
```java
    org.apache.kafka.connect.errors.DataException: Unknown Java type for schemaless data: class org.apache.kafka.connect.data.Struct
```

when trying to extract a simepl SQLITE table. I am using the the 0.9 KAFKA Snapshot and the Confluent 2.0 SNAPSHOT

Any idea? Lib mismatches?

Thank you,
Artyom

**Comment by ewencp** [comment]
@Arttii This was due to some changes in the upstream Kafka Connect API. The commit https://github.com/confluentinc/kafka-connect-jdbc/commit/0ed27e089a220709b87ca85da5c86e78b408be83 which is included in #16 should fix this.

**Comment by Arttii** [comment]
Hi,

Thanks. I noticed the new branch like an hour later. It works fine now.
